### DISCUSSION TITLE
Translations from database are only returned if not present in file

### DISCUSSION THREAD

**Discussion by leonardobuares** [spatie/laravel-translation-loader#155] [Bugs]
The readme says: "If a translation is present in both a file and the database, the database version will be returned." but the opposite happens to me. I'm using json files, I tried in a new laravel 9 + php 8.0 project and same thing happens. Is this a bug or am I doing something wrong? Even creating a new laravel project I can't get the database version as priority.

**Comment by thomasbuckle-uk** [comment]
I've also gotten this, any way to resolve in Laravel 10?
